R/tuneEDA.R

```diff
@@ -0,0 +1,62 @@
+tuneEDA = function(learner, task, resampling, measures, par.set, control, opt.path, show.info) {
+  requirePackages("copulaedas", why = "tuneEDA", default.method = "load")
+
+  low = getLower(par.set)
+  upp = getUpper(par.set)
+  start = control$start
+
+  if (is.null(start))
+    start = sampleValue(par.set, start, trafo = FALSE)
+  start = convertStartToNumeric(start, par.set)
+
+  ctrl.eda = list(popSize = 100L, maxit = 50L, type = "UMDA")
+  ctrl.eda = insert(ctrl.eda, control$extra.args)
+  cx = function(x, par.set) convertXNumeric(x, par.set)
+
+  if (is.null(ctrl.eda$type) || ctrl.eda$type %nin% c("UMDA", "GCEDA", "CVEDA", "DVEDA"))
+    ctrl.eda$type = "UMDA"
```

Comment:
This seems a bit strange, we fallback to the default if the user specifies nothing or a "wrong" type. why not just setting a default and doing a check if the user sets something stupid and throw an error